addons/uav/CfgVehicles.hpp

```diff
@@ -0,0 +1,441 @@
+
+class CBA_Extended_EventHandlers;
+class UAVpod;
+class UAVoptics;
+
+class CfgVehicles {
+    class B_UAV_01_F;
+    class ACE_B_UAV_01_F: B_UAV_01_F {
+        magazines[] += {"laserbatteries"};
```

Comment:
I think it will be clearer to just define the whole array here instead of using `+=`

Reply:
But that would overwrite mags that other mods might add to B_UAV_01_F, which these copy should also work with

Reply:
```suggestion
        magazines[] += {"Laserbatteries"};
```